### ISSUE TITLE
Got error when installing: `File not found with singular glob: ... organizations/organizations.js` 

### ISSUE THREAD

**Issue by zwcloud** [Datasilk/Kandu#12]
```
root@localhost:~/kandu# git submodule
 dd3a43087783e90d69496a7ea987ffa97b3153f1 App/CSS/tapestry (dd3a430)
 996ac3dbff28e091e929ccd085b2ccfa4c1edffb App/Common/Utility/Malicious (heads/master)
 ca39fc086c1b4b4ad1fb6f6e6b1c46cf97599b0a App/Scripts/platform (ca39fc0)
 a77cb1a29013953130283de323da6bc751a0e311 App/Scripts/selector (0.9.12-46-ga77cb1a)
 a8877009720f52604ee51f9539026ed1740397f2 Query/Core (heads/master)
root@localhost:~/kandu# gulp default
[04:53:59] Using gulpfile ~/kandu/gulpfile.js
[04:53:59] Starting 'default'...
[04:53:59] Starting 'js'...
[04:53:59] Starting 'js:app'...
[04:54:00] Finished 'js:app' after 221 ms
[04:54:00] Starting 'js:selector'...
[04:54:00] Finished 'js:selector' after 25 ms
[04:54:00] Starting 'js:platform'...
[04:54:00] Finished 'js:platform' after 45 ms
[04:54:00] Starting 'js:utility'...
[04:54:00] Finished 'js:utility' after 17 ms
[04:54:00] Starting 'js:dashboard'...
[04:54:00] 'js:dashboard' errored after 8.42 ms
[04:54:00] Error: File not found with singular glob: /root/kandu/App/views/organizations/organizations.js (if this was purposeful, use `allowEmpty` option)
    at Glob.<anonymous> (/root/kandu/node_modules/glob-stream/readable.js:84:17)
    at Object.onceWrapper (events.js:286:20)
    at Glob.emit (events.js:198:13)
    at Glob.EventEmitter.emit (domain.js:448:20)
    at Glob._finish (/root/kandu/node_modules/glob/glob.js:197:8)
    at done (/root/kandu/node_modules/glob/glob.js:182:14)
    at Glob._processSimple2 (/root/kandu/node_modules/glob/glob.js:688:12)
    at /root/kandu/node_modules/glob/glob.js:676:10
    at Glob._stat2 (/root/kandu/node_modules/glob/glob.js:772:12)
    at lstatcb_ (/root/kandu/node_modules/glob/glob.js:764:12)
[04:54:00] 'js' errored after 322 ms
[04:54:00] 'default' errored after 326 ms
```

**Comment by zwcloud** [comment]
I'm trying to fix this. I'll appreciate it if you give some hint. @markentingh

**Comment by zwcloud** [comment]
Oh, it seems to be caused by letter cases. Path on Linux is case-sensitive. The file does exists, but not recognized by Linux.

~/kandu/App/Views/Organizations/organizations.js
~/kandu/App/views/organizations/organizations.js

**Comment by zwcloud** [comment]
Fixed in #13, @markentingh  please review
